spec/index.html

```diff
@@ -656,41 +656,51 @@ <h4>The <code>&lt;<dfn id="the-map-element">map</dfn>&gt;</code> element</h4>
         </dd>
         </dl>
-        <p>The <code id="dom-areas">areas</code> attribute must return an <code>HTMLCollection</code> 
+        
+        <p>The <code id="dom-areas">areas</code> attribute must return an <a href="https://dom.spec.whatwg.org/#htmlcollection"><code>HTMLCollection</code></a> 
         rooted at the <a href="#the-map-element"><code>map</code></a> element, whose filter matches only  <a href="#the-area-element"><code>area</code></a> child elements.</p>
 
-        <p>The <code id="dom-layers">layers</code> attribute must return an <code>HTMLCollection</code> 
+        <p>The <code id="dom-layers">layers</code> attribute must return an <a href="https://dom.spec.whatwg.org/#htmlcollection"><code>HTMLCollection</code></a> 
         rooted at the <a href="#the-map-element"><code>map</code></a> node, whose filter matches only <a href="#the-layer-element"><code>layer</code></a> child elements of this <a href="#the-map-element"><code>map</code></a> element.</p>
+        
+        <p>The <dfn id="attr-map-zoom"><code>zoom</code></dfn> attribute indirectly identifies an initial scale of the map.</p>
+        <p>The <a href="#dom-htmlmapelement-zoom"><code>zoom</code></a> IDL attribute must <a href="https://html.spec.whatwg.org/multipage/common-dom-interfaces.html#reflect">reflect</a> the content attribute of the same name.</p>
+        
+        <p>The <dfn id="attr-map-lat"><code>lat</code></dfn></a> and <dfn id="attr-map-lon"><code>lon</code></dfn> attributes locate the initial center of the map.</p>
+        <p>The <a href="#dom-htmlmapelement-lat"><code>lat</code></a> and <a href="#dom-htmlmapelement-lon"><code>lon</code></a> IDL attributes must <a href="https://html.spec.whatwg.org/multipage/common-dom-interfaces.html#reflect">reflect</a> the content attribute of the same name.</p>
 
-        <p>The <code>controls</code> boolean attribute indicates whether default map controls should be displayed on the map.  The nature and position of the controls 
-        is user agent implementation-defined.  The controls can be added or removed by toggling the controls property of the element.
-        </p>
-        <p>The <code>projection</code> attribute value identifies the coordinate system for the map and all the <a href="#the-layer-element"><code>layer</code></a> element children, with the exception of
+        <p>The location and zoom level of the map created with the <a href="#the-map-element"><code>map</code></a> element can be changed via the <code>zoomTo()</code> API method, which will change the <code>zoom</code>, <code>lat</code> and <code>lon</code> properties.</p>
+
+        <p>The <dfn id="attr-map-controls"><code>controls</code></dfn> attribute is a <a href="https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#boolean-attribute">boolean attribute</a>. If present, it indicates that the author has not provided a scripted controller and would like the user agent to provide its own set of controls.</p>
```

Comment:
> If present, it indicates that the author has not provided a scripted controller

Suggested: "If present, it indicates that the author has not provided a scripted layer control"

Sometimes "controller" might be confused with the same term from "model-view-controller", so should avoid that connotation.

Reply:
That paragraph is about controls in general, so saying "scripter layer control" wouldn't make sense. Note that the text is a straight copy paste from https://html.spec.whatwg.org/multipage/media.html#user-interface

Reply:
"has not provided scripted controls", would that be better?

Reply:
Good

Reply:
Done.